src/sentry/templates/sentry/account/recover/index.html

```diff
@@ -22,4 +22,5 @@ <h3>{% trans "Recover Account" %}</h3>
       <button type="submit" class="btn btn-primary">{% trans "Send Email" %}</button>
     </fieldset>
   </form>
+  <p>{% blocktrans %}Need help with your account?  <a href="https://help.sentry.io/"> Click here to check out our help center.</a>{% endblocktrans %}</p>
```

Comment:
```suggestion
  <p>
  {% blocktrans %}
  Need help with your account? Click <a href="https://help.sentry.io/">here</a> to check out our help center.
  {% endblocktrans %}
  </p>
```

nit: formatting since i don't think we have automatic html formatting